### PR TITLE
Add dummy error handler for proxies

### PR DESCRIPTION
**Pull request by alltouch**
In our project we have following error:
client aborting request to proxy => proxy raises error => test fails => we unhappy

The only solution i know add error handler. Implemented simple code for it.
@johanneswuerbach, what do you think about it?

### CODE REVIEW

File: lib/server/index.js [hunk 1]
```diff
@@ -187,6 +187,13 @@ Server.prototype = {
     if (proxies) {
       self.proxy = new httpProxy.createProxyServer({changeOrigin: true});
 
+      self.proxy.on('error', function (err, req, res) {
+        res.writeHead(500, {
+          'Content-Type': 'text/plain'
+        });
+
+        res.end('Something went wrong. And we are reporting a custom error message.');
```
Comment (johanneswuerbach) [inline lib/server/index.js]: Feels like we should also log the error itself. Maybe sentry.captureErorror?

Comment (akatov) [inline lib/server/index.js]: where does testem use sentry?

Comment (johanneswuerbach) [inline lib/server/index.js]: Sorry, I got confused. Logging the error itself and the stack should be fine for now. As testem shouldn't be used in production the stack can also be part of the response for easier development

**Comment by alltouch** [comment]
@johanneswuerbach
Updated code. Do we need any other changes?

**Comment by johanneswuerbach** [comment]
Sorry for letting this linger so long, would you mind adding a test here https://github.com/testem/testem/blob/master/tests/server_tests.js? Then this is ready to merge.

**Pull request closed without merging**